azure-pipelines.yml

```diff
@@ -143,7 +143,7 @@ jobs:
         - "/sw/***"
 
 - job: cw310_sw_build
-  displayName: Build Software required for CW310 FPGA synthesis
+  displayName: Build Software required for CW310 FPGA synthesis with meson
```

Comment:
Maybe: This is true and correct but it's probably just going to create a merge conflict with https://github.com/lowRISC/opentitan/pull/12695

Reply:
No worries, this will need to be rebased on those changes any way so I will wait until #12695 merges.